rust/operator-binary/src/controller.rs

```diff
@@ -265,8 +319,27 @@ fn build_server_rolegroup_daemonset(
         .image(image)
         .command(build_opa_start_command())
         .add_env_vars(env)
-        .add_container_port("http", APP_PORT.into())
+        .add_container_port(APP_PORT_NAME, APP_PORT.into())
         .add_volume_mount("config", "/stackable/config")
+        .readiness_probe(Probe {
+            initial_delay_seconds: Some(10),
+            period_seconds: Some(10),
+            failure_threshold: Some(5),
+            tcp_socket: Some(TCPSocketAction {
+                port: IntOrString::String(APP_PORT_NAME.to_string()),
+                ..TCPSocketAction::default()
+            }),
```

Comment:
Any reason we're not doing a HTTP probe here?

Reply:
done